### ISSUE TITLE
[Quest][Northrend] Mutiny on the Mercy

### ISSUE THREAD

**Issue by evilstein** [dalaranwow/dalaran-wow#457]
The NPCs that drop the requested items are bellow the ship, but still move with it. This makes it nearly impossible to kill them and loot them.
Quest link: http://www.wowhead.com/quest=11527

**Comment by velare** [comment]
Quest doesn't appear to be doable any longer, I checked both (37,71) and all along the ship's pat route(with sense undead and a /tar macro) and couldn't find the mobs in question.

**Comment by ashido96** [comment]
NPC's are missing from the boat and the location, uncompletable.

**Comment by irodeyourmom** [comment]
Quest currently incomplete. No way to obtain the barrels.

**Comment by deltafoxnine** [comment]
At first the mobs wasn't down stairs, i waited some minutes, then they spawned.
Seems to be a lot of pathing problems in the lower deck. My hunter pet wasn't able to move around down there, and i had to to the opposite side of the stairs to kill them(still a lot of pathing problems on the mobs, but it was doable) I think as ranged class, it is much more difficult because you are depend on correct pathing.

**Comment by Arrowzinger** [comment]
Still an Issue, I'd make a new post to get more attention but they always get taken down.

**Comment by thezk** [comment]
Can't find any ghoul on the ship nor the point, and they are not spawning, it should be still bugged.

**Comment by Pengwindk** [comment]
The npcs do not appear on the Mercy or on the chunk of ice, or any were near the given coordinates.

**Comment by Augur89** [comment]
Yep, you're all right. The quest was doable back then but today I went there with my alt and didn't find any mob. This quest never worked as intended but it got a temporary solution and was doable; it seems that with the big patch that fixed a lot of quests this one got no attention and the temporary fix was scraped, thus rendering it uncompletable nowadays.

**Comment by MrAndersonX** [comment]
Confirmed - still bugged. Needs a fix.

**Comment by sklivvz** [comment]
The extra ones to complete the quest are now at 35, 72 on a floating piece of ice. The ones that are supposed to be on the ship *fell down* at the bottom of the sea at 36, 72 and cannot be aggro'd (not in line of sight).

See ![the ones fallen down](http://i.imgur.com/G7dUeO4.jpg) and ![the killable ones](http://i.imgur.com/nP97wtn.jpg)

**Comment by DrFuler** [comment]
[01/03/2016] - Still bugged, but the temporary fix makes it doable.

**Comment by Rushor** [comment]
this is related to creatures and transport - general problem

**Comment by Rushor** [comment]
fixed on ptr from trinity http://i.imgur.com/lxY1j9Q.jpg

**Comment by TeridanDW** [comment]
Sadly this is not working at all, creatures look very good the first time you go there (after server start), but after you kill them once they do not respawn inside the ship, but someone else. I can't even find where they respawn, so we are again at point 1.

**Comment by TeridanDW** [comment]
Update: after some time these creatures did respawn inside the boat, i don't know what trigger it. I was doing something else in diffrent part of the world and when i came back after ~30 mins there were there.

**Comment by fvicaria** [comment]
@Rushor: would you mind sharing the fix? I have the same issue. :)

**Comment by Radschpud** [comment]
not working. no undead creatures near to drop the barrels.

![wowscrnshot_120717_141102](https://user-images.githubusercontent.com/14928789/33717272-9025f41e-db59-11e7-96db-fcf0e8de2063.jpg)

**Comment by Shreduction** [comment]
I can confirm that this quest is still not working...

**Comment by Shreduction** [comment]
Update: Mobs are spawned on the location mentioned above and lootable.

**Comment by Technicz** [comment]
Mutinous Sea Dog NPC's do spawn, both above and below the water.  The previous comments about the line of sight issue still exist for the ones under the water.  There are a group of 5 Mutinous Sea Dogs that can be used to complete this quest on a block of ice in the water at coordinates 35, 72.

Therefore, this seems "broken" in the sense that the NPC's aren't spawning in their designed positions, but this quest can be completed.  Details via screenshots are below:

- Enemies on the ice block at 35, 72

![iceblock2](https://user-images.githubusercontent.com/46239286/50535397-88df4e00-0b17-11e9-8c81-5ac626e6be44.png)


- Ice block in a different view post-clearing the enemies

![iceblock](https://user-images.githubusercontent.com/46239286/50535402-94327980-0b17-11e9-8f54-f9abc98cc314.png)


- In-game map location if coordinates of 35, 72 are unhelpful

![map](https://user-images.githubusercontent.com/46239286/50535409-a7454980-0b17-11e9-9dfc-12dc01c5f01a.png)


- In-game quest item tooltip proof

![questitem](https://user-images.githubusercontent.com/46239286/50535420-ca6ff900-0b17-11e9-9005-895cc8cf0864.png)


- In-game quest completion proof

![questcomplete](https://user-images.githubusercontent.com/46239286/50535423-d2c83400-0b17-11e9-9ad6-a1c6d1c166e9.png)


Hope this helps!
